Makefile.toml

```diff
@@ -155,10 +158,7 @@ script = ["""
   rm -f ios/Classes/XaynAiFFiDart.h
   rm -f lib/src/common/ffi/genesis.dart
   rm -f lib/src/mobile/ffi/genesis.dart
-  rm -f example/assets/wasm_bindings/genesis_bg.wasm
-  rm -f example/assets/wasm_bindings/genesis.js
-  rm -f example/assets/wasm_bindings/package.json
-  rm -rf example/assets/wasm_bindings/snippets/
+  find $EXAMPLE_ASSETS_DIR/wasm_bindings -type f ! -name .gitkeep -exec sh -c "rm {}" \\;
```

Comment:
this crashes due to the `cd "${DART_WORKSPACE}"` at the beginning of the script, it could be fixed like
```sh
[tasks.clean-bindgen]
script = ["""
  rm -f $DART_WORKSPACE/ios/Classes/XaynAiFFiCommon.h
  rm -f $DART_WORKSPACE/ios/Classes/XaynAiFFiDart.h
  rm -f $DART_WORKSPACE/lib/src/common/ffi/genesis.dart
  rm -f $DART_WORKSPACE/lib/src/mobile/ffi/genesis.dart
  find $EXAMPLE_ASSETS_DIR/wasm_bindings -type f ! -name .gitkeep -exec sh -c "rm {}" \\;
"""]
```

Reply:
thanks! fixed